ctk/common/src/main/kotlin/org/codice/compliance/verification/core/internal/StatementVerifier.kt

```diff
@@ -113,9 +118,30 @@ internal class StatementVerifier(val node: Node) {
                     || (friendlyNameAttr != null && friendlyNameAttr.textContent == null)) {
                 verifyAttributeValue(it)
             }
+
+            for (i in it.attributes.length - 1 downTo 0) {
```

Comment:
✏️ Sort-of related to the above comment: In an upcoming PR I'm adding an `attributeList()` extension function to the `Node` class in `Common.kt`.

```
fun Node.attributeList(): List<Node> {
    val attributesList = mutableListOf<Node>()
    this.attributes.let {
        for (i in it.length downTo 0) {
            attributesList.add(it.item(i))
        }
    }
    return attributesList
}
```
If you want you can add that now and stream the attributeList instead.